eZ/Bundle/EzPublishCoreBundle/ApiLoader/StorageEngineFactory.php

```diff
@@ -65,20 +65,22 @@ public function getStorageEngines()
      */
     public function buildStorageEngine()
     {
-        $repositoryConfig = $this->storageRepositoryProvider->getRepositoryConfig();
+        $repositoryConfig = $this->repositoryConfigurationProvider->getRepositoryConfig();
 
         if (
         !(
-            isset( $repositoryConfig['engine'] )
-            && isset( $this->storageEngines[$repositoryConfig['engine']] )
+            isset( $repositoryConfig['storage']['engine'] )
+            && isset( $this->storageEngines[$repositoryConfig['storage']['engine']] )
         )
         )
```

Comment:
Yes, not introduced here though. Fixed in https://github.com/ezsystems/ezpublish-kernel/commit/ea4b9120bbd0e3ec67407364bca506d0543f11ca.